internal/pkg/deploy/cloudformation/testdata/rendered_pipeline_cfn_template.yml

```diff
@@ -0,0 +1,378 @@
+            Effect: Allow
+            Action:
+              - sts:AssumeRole
+            Resource:
+              - arn:aws:iam::786043277641:role/chickenProject-test-EnvManagerRole
+              - arn:aws:iam::786043277641:role/chickenProject-prod-EnvManagerRole
+      Roles:
+        -
+          !Ref PipelineRole
+  Pipeline:
+    Type: AWS::CodePipeline::Pipeline
+    DependsOn:
+      - PipelineRole
+      - PipelineRolePolicy
+    Properties:
+      ArtifactStores:
+        -
+          Region: us-east-1
+          ArtifactStore:
+            Type: S3
+            Location: chicken-us-east-1
+            EncryptionKey:
+              Id: arn:aws:kms:us-east-1:620297136107:key/30131d3f-c30f-4d49-beaa-cf4bfc07f34e
+              Type: KMS
+        -
+          Region: us-west-2
+          ArtifactStore:
+            Type: S3
+            Location: chicken-us-west-2
+            EncryptionKey:
+              Id: arn:aws:kms:us-west-2:620297136107:key/80de5f7f-422d-4dff-8f4d-01f6ec5715bc
+              Type: KMS
+        -
+          Region: us-west-1
+          ArtifactStore:
+            Type: S3
+            Location: chicken-us-west-1
+            EncryptionKey:
+              Id: arn:aws:kms:us-west-1:620297136107:key/75668c57-ec4b-4d0c-b880-8dc3fa78f6d1
+              Type: KMS
+      RoleArn: !GetAtt PipelineRole.Arn
+      Name: !Ref AWS::StackName
+      Stages:
+        - Name: Source
+          Actions:
+            - Name: SourceCodeFor-chickenProject
+              ActionTypeId:
+                Category: Source
+                Owner: ThirdParty
+                Version: 1
+                Provider: GitHub
+              Configuration:
+                Owner: hencrice
+                Repo: amazon-ecs-cli-v2
+                Branch: master
+                OAuthToken: !Sub
+                    - '{{resolve:secretsmanager:${SecretId}}}'
+                    - { SecretId: !Ref GitHubOAuthSecretId }
+              OutputArtifacts:
+                - Name: SCCheckoutArtifact
+              RunOrder: 1
+        -
+          Name: Build
+          Actions:
+          -
+            Name: Build
+            ActionTypeId:
+              Category: Build
+              Owner: AWS
+              Version: 1
+              Provider: CodeBuild
+            Configuration:
+              ProjectName: !Ref BuildProject
+            RunOrder: 1
+            InputArtifacts:
+              - Name: SCCheckoutArtifact
+            OutputArtifacts:
+              - Name: BuildOutput
+        
+        
+        - Name: DeployTochickenProject-test
+          Actions:
+            - Name: CreateChangeSet-frontend-test
+              Region: us-west-2
+              ActionTypeId:
+                Category: Deploy
+                Owner: AWS
+                Version: 1
+                Provider: CloudFormation
+              Configuration:
+                ChangeSetName: chickenProject-frontend-test
+                ActionMode: CHANGE_SET_REPLACE
```

Comment:
Will this *create* the stack if it doesn't exist?

Reply:
yep: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/continuous-delivery-codepipeline-action-reference.html

Reply:
yep